languages/python/example.py

```diff
@@ -0,0 +1,47 @@
+import json
+import logging
+import sys
+from BitwardenClient.bitwarden_client import BitwardenClient
+from BitwardenClient.schemas import client_settings_from_dict, DeviceType
+
+# Create the BitwardenClient, which is used to interact with the SDK
+client = BitwardenClient(client_settings_from_dict({
+    "apiUrl": "http://localhost:4000",
+    "deviceType": DeviceType.SDK,
+    "identityUrl": "http://localhost:33656",
+    "userAgent": "Python",
+}))
+
+# Add some logging & set the org id
+logging.basicConfig(level=logging.DEBUG)
+organization_id = "org_id_here"
+
+# Attempt to authenticate with the Secrets Manager Access Token
+result = client.access_token_login("access_token_here")
+
+if result.success == False:
```

Comment:
This check should be applied to all commands, I think the best way would be to update `BitwardenClient`:

```py
    def _run_command(self, command: Command) -> Any:
        response_json = self.inner.run_command(json.dumps(command.to_dict()))
        response = json.loads(response_json)

        if response["success"] == False:
            raise Exception(response["errorMessage"])

        return response
```

That way it won't be needed to be handled by clients directly, they'll just get an exception.